From 19f5cd7a63708736cac80394cecc9408d88a289a Mon Sep 17 00:00:00 2001
From: Richard Elberger <elberger@amazon.com>
Date: Thu, 9 Apr 2026 14:42:33 -0400
Subject: [PATCH] Disable strip in Cargo profile

Let Yocto handle stripping instead of Cargo doing it during
compilation.

Upstream-Status: Inappropriate [Yocto-specific]
---
 rust/Cargo.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/rust/Cargo.toml b/rust/Cargo.toml
index e28b9e8..f470e13 100644
--- a/rust/Cargo.toml
+++ b/rust/Cargo.toml
@@ -30,7 +30,6 @@ bindgen = "0.72.1"
 cc = "1"
 
 [profile.release]
-strip = true
 opt-level = "z"
 lto = true
 codegen-units = 1
